sources/api/schnauzer/src/lib.rs

```diff
@@ -115,6 +115,7 @@ pub fn build_template_registry() -> Result<handlebars::Handlebars<'static>> {
     template_registry.register_helper("join_map", Box::new(helpers::join_map));
     template_registry.register_helper("default", Box::new(helpers::default));
     template_registry.register_helper("ecr-prefix", Box::new(helpers::ecr_prefix));
+    template_registry.register_helper("join-array", Box::new(helpers::join_array));
```

Comment:
nit: `join_array` would be more consistent.  (I wish we had caught `ecr-prefix`.)